### ISSUE TITLE
Ideas

### ISSUE THREAD

**Issue by tigert** [legoboyvdlp/Finland-fg-CustomScenery#1]
Hi. Been a while since I was away from sim stuff, but since you have pushed this to git, I wanted to see how to improve it. 

We have official mast and other obstacle data here:
https://www.ais.fi/en/products-and-services/air-navigation-obstacles

This could perhaps be used with some custom macro -like things that would generate procedurally some masts to a given height, maybe using a few types of models as pieces for various sizes of masts and chimneys? Also adding obstacle avoidance lights etc..

I am not 100% sure whether some of these already exist in fg scenery, but this would add a few useful VFR landmarks nevertheless.

**Comment by legoboyvdlp** [comment]
Hi,
Might be best to put this on the flightgear devel list and do this everywhere, not just Finland?

thanks for the ideas!

**Comment by tigert** [comment]
Yeah. Although that obstacle listing is only for Finland, but such macro thingy could be useful for global scenery if there is data for it.

I assume this scenery work you did is placing objects on airports, or do you do something with the landclass / mesh? There's this on the forum: https://forum.flightgear.org/viewtopic.php?f=5&t=21372&p=194563 but looks like it's not very alive right now. I might be interested in helping with this stuff, but I am not very familiar with the scenery tools. I guess I will poke at that thread and see whether anything is going on there. 

Is mesh / landclass stuff something that should generally be done for the global online scenery and not as a custom thing?

(sorry for many questions :-))

**Comment by legoboyvdlp** [comment]
I gathered data from various sources and ran the TerraGear tools over it. TerraSync in the future will probably use the exact same sources I used for version 3.0 of world scenery. That is being worked on - it's still not going to be soon, but its closer than it was a few years ago!

**Comment by tigert** [comment]
Nice! 

Hmm, sorry to throw another url at you, but I wonder what this is...? https://www.avoindata.fi/data/en_GB/dataset/vantaan-3d-rakennukset

I intend to help down the road where I can, but just found that and I need to find time to look at this a bit more, but figured it might be interesting.

(Vantaa is near and around EFHK if you need a reference)
